src/App.js

```diff
@@ -20,14 +21,42 @@ const StyledAppContainer = styled.div`
   height: 100vh;
 `
 
+const getTrailData = (trailFilename) => {
+  return fetch(`data/trails/${trailFilename}`).then((res) => res.json());
+};
+
 
 function App() {
+  const [selectedTrail, setSelectedTrail] = useState({
+    type: "FeatureCollection",
+    features: [
+      {
+        type: "Feature",
+        geometry: {
+          type: "MultiLineString",
+          coordinates: [[]],
+          bbox: [],
+        },
+        properties: {
+          name: "Trail Planner Map",
+          desc: "Generated by AllTrails https://www.alltrails.com/",
+        },
+      },
+    ],
+  });
```

Comment:
instead of creating a placeholder geojson object, your component should be handle an initial `undefined` or `{}` state